src/components/ProfileBox/AddNewProfile.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+
+import styles from './ProfileBox.module.scss';
+
+import Plus from '#src/icons/Plus';
+
+type Props = {
+  onClick: () => void;
+};
+
+const AddNewProfile = ({ onClick }: Props) => {
+  const { t } = useTranslation('user');
+  return (
+    <div onClick={onClick} className={classNames(styles.wrapper, styles.addProfileContainer)}>
```

Comment:
When using `<div>` as button, you should also make it keyboard accessible:

```suggestion
  const { t } = useTranslation('user');
  const keyDownHandler = (event: React.MouseEvent<HTMLDivElement>) => (event.key === 'Enter' || event.key === ' ') && onClick()

  return (
    <div onClick={onClick} tabIndex="0" onKeyDown={keyDownHandler} className={classNames(styles.wrapper, styles.addProfileContainer)}>
```